### DISCUSSION TITLE
mac vscode编译报错: ld terminated with signal 9 [Killed]

### DISCUSSION THREAD

**Discussion by wangpeix** [alibaba/ilogtail#1409] [Help] [answered]
ilogtail版本: 2.0.0
开发环境： mac + vscode
Debug编译报错： ld terminated with signal 9 [Killed]
查了下，可能是mac docker 资源给的太少，但资源已经给的挺多的了，如下:
<img width="1265" alt="image" src="https://github.com/alibaba/ilogtail/assets/105137931/ba778a8f-375f-4545-973d-5147a3a7e174">

大佬们帮看下~

**Comment by yyuuttaaoo** [comment, marked as answer]
你好，使用的是哪个编译命令编译的？如果不是在开发容器内编译直接make all或者make core的话，可以设置一下环境变量MAKE_JOBS={x}，x小一些。如果是容器内编译，那么make -sj{x}的时候x小一些。

**Reply by wangpeix** [reply]
在容器内编译，我试一下，感谢
